src/non-profits.html

```diff
@@ -78,8 +78,39 @@ <h2 class="section-heading">Do you have any questions?</h2>
 <section>
   <h2 class="section-heading">Start your journey here</h2>
   <p>Please fill in the form below to see how we can help bring your project to life.
-    <p>
-      <a href="non-profits-contact.html" class="button">Work with us</a>
+  <form method="POST" action="//formspree.io/help@womenhackfornonprofits.com">
+    <input type="hidden" name="_subject" value="New Submission: New Non-Profit Project Application" />
+    <div class="form-group">
+      <label for="non-profit-name">Name of your Non-Profit</label>
+      <input type="text" name="name" placeholder="Code 2040" required aria-required="true" class="form-control">
+    </div>
+    <div class="form-group">
+      <label for="contact-name">Contact Name</label>
+      <input type="text" name="contact-name" placeholder="Laura Weidman Powers" required aria-required="true" class="form-control">
+    </div>
+    <div class="form-group">
+      <label for="email">Email</label>
+      <input type="email" name="_replyto" placeholder="Laura@example.com" required aria-required="true" class="form-control">
+    </div>
+    <div class="form-group">
+      <label for="contact-tel">Phone <span class="help-text">(optional)</span></label>
+      <input type="tel" name="contact-tel" placeholder="07912 345 678" class="form-control">
+    </div>
+    <div class="form-group">
+      <label for="project-description">What kind of technical help would you like?</label>
+      <textarea name="project-description" rows="3" placeholder="Website, mobile, app" class="form-control"></textarea>
```

Comment:
Think `app` can be misleading. To a client an `app` could mean a mobile application, but to a techie that could mean a desktop app, web app or mobile app. Think it's best to explicitly say what you mean to reduce potential confusion